packages/editor/src/components/collapsible-block-toolbar/style.scss

```diff
@@ -74,9 +74,10 @@
 
 	&.is-collapsed {
 		display: none;
+
+		~ .editor-collapsible-block-toolbar__toggle {
+			margin-left: $grid-unit-10;
+		}
```

Comment:
Adds a "gap" equal to that of the document tools toolbar. This was previously by a margin on that toolbar that I changed it because it didn’t always need to be there. There’s more info in a comment on that change.